pkg/ride/tree_evaluation_test.go

```diff
@@ -3716,7 +3730,12 @@ func TestRecipientAddressToString(t *testing.T) {
 			Attachment:  nil,
 		},
 	}
-	obj, err := transactionToObject(tree.LibVersion, proto.TestNetScheme, false, tx)
+	me := &mockRideEnvironment{
+		schemeFunc:                         func() byte { return proto.TestNetScheme },
+		libVersionFunc:                     func() (ast.LibraryVersion, error) { return tree.LibVersion, nil },
+		consensusImprovementsActivatedFunc: func() bool { return false },
+	}
+	obj, err := transactionToObject(me, tx)
 	require.NoError(t, err)
 	env := newTestEnv(t).withLibVersion(tree.LibVersion).withComplexityLimit(tree.LibVersion, 2000).
 		withMessageLengthV3().withTransactionObject(obj)
```

Comment:
I think better to reimplement builder function `withTransactionObject` and replace it with something like `withTransaction` and hide this environment manipulations  inside of this method.